app/src/main/java/com/lorenzovainigli/foodexpirationdates/view/composable/MyBottomAppBar.kt

```diff
@@ -48,7 +49,7 @@ fun MyBottomAppBar(
         unselectedIcon = Icons.Outlined.Info
     )
     )
-    NavigationBar {
+    NavigationBar(containerColor = MaterialTheme.colorScheme.inverseOnSurface) {
```

Comment:
It's correct to change `containerColor`, but please assign the following value:
```suggestion
    NavigationBar(
        containerColor = MaterialTheme.colorScheme.surfaceColorAtElevation(TonalElevation.level2())
    ) {
```

Reply:
ok I will check